### DISCUSSION TITLE
ScriptReference/MonoBehaviour.OnEnable

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#396] [Notes]
# ScriptReference/MonoBehaviour.OnEnable

https://docs.unity3d.com/ScriptReference/MonoBehaviour.OnEnable.html

**Comment by Hunter-Bobeck** [comment]
Description is misleading; **objects** cannot "*become enabled*", rather **MonoBehaviours** can.  
These functions are called when either:  
- MonoBehaviour.enabled becomes true while it is active in the hierarchy.  
- The MonoBehaviour becomes active in the hierarchy while it is enabled.
